src/lib/x509/certstor_system_linux/certstor_linux.cpp

```diff
@@ -0,0 +1,18 @@
+/*
+* Certificate Store
+* (C) 1999-2019 Jack Lloyd
+* (C) 2019      Patrick Schmidt
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/build.h>
+
+#include <botan/certstor_linux.h>
+
+namespace Botan {
+Certificate_Store_Linux::Certificate_Store_Linux() :
+   Flatfile_Certificate_Store(BOTAN_LINUX_CERTSTORE_DEFAULT_FILE)
+   {
+   }
+}
```

Comment:
I believe this should check for `#if defined(BOTAN_TARGET_OS_HAS_FILESYSTEM)` otherwise it won't compile without a file system, because the header would be empty